tools/kubequery/kubequery-fleet.yml

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) 2020-present, The kubequery authors
+# Copyright (c) 2022-present, Fleet
+#
+# This source code is licensed as defined by the LICENSE file found in the
+# root directory of this source tree.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+
+# Adapted from original provided by Kubequery project.
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubequery
+  labels:
+    app.kubernetes.io/name: kubequery
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubequery-sa
+  namespace: kubequery
+  labels:
+    app.kubernetes.io/name: kubequery-sa
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubequery-clusterrole
+  labels:
+    app.kubernetes.io/name: kubequery-clusterrole
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+rules:
+- apiGroups: ["", "admissionregistration.k8s.io", "apps", "autoscaling", "batch", "events.k8s.io", "networking.k8s.io", "policy", "rbac.authorization.k8s.io", "storage.k8s.io"]
+  resources: ["*"]
+  verbs: ["get", "list", "watch"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kubequery-clusterrolebinding
+  labels:
+    app.kubernetes.io/name: kubequery-clusterrolebinding
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+roleRef:
+  kind: ClusterRole
+  name: kubequery-clusterrole
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: kubequery-sa
+  namespace: kubequery
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubequery-config
+  namespace: kubequery
+  labels:
+    app.kubernetes.io/name: kubequery-config
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+data:
+  enroll.secret: Y3nHXcZUBcFJc/7e6V6k1z7RG22rrAnQ
```

Comment:
This is not known ahead of time. We should include a note in the README.md about fetching it and changing it here.